lib/app/utils/language.dart

```diff
@@ -53,6 +53,11 @@ class AppTranslations extends Translations {
           'Create alarm': 'Create alarm',
           'Join alarm': 'Join alarm',
           'Okay': 'Okay',
+          'Yes': 'Yes',
```

Comment:
You'll have to add the translations for these as well.

Reply:
ok. i will do it